README.md

```diff
@@ -44,14 +44,14 @@ plugin_directory = "path/to/plugin/directory"
 ```
 
 ```sh
-$ vault write sys/plugins/catalog/kerberos-auth-plugin sha_256="$(shasum -a 256 'vault-plugin-auth-kerberos' | cut -d ' ' -f1)" command="vault-plugin-auth-kerberos -client-cert server.crt -client-key server.key"
+$ vault write sys/plugins/catalog/auth/kerberos sha_256="$(shasum -a 256 'vault-plugin-auth-kerberos' | cut -d ' ' -f1)" command="vault-plugin-auth-kerberos -client-cert server.crt -client-key server.key"
 ```
 
 2. Enable the Kerberos auth method:
 
 ```sh
-$ vault auth-enable -path=kerberos -plugin-name=kerberos-auth-plugin -passthrough-request-headers=Authorization plugin
-Successfully enabled 'kerberos' at 'kerberos'!
+$ vault auth enable -passthrough-request-headers=Authorization kerberos
+Success! Enabled kerberos auth method at: kerberos/
```

Comment:
Could you keep the `-passthrough-request-headers=Authorization`?

Passing through the Auth header was broken by vault ~1.0, but works again now so it'd be good to have it in there.

Reply:
Yeah absolutely! It's still there on line 53, and on line 53, that's simply the response output that Vault gives. I can strip that second line though?

Reply:
Looks great!